qemu/tests/numa_basic.py

```diff
@@ -23,24 +22,27 @@ def run(test, params, env):
     :param params: Dictionary with the test parameters
     :param env: Dictionary with test environment.
     """
-    error.context("Get host numa topological structure", logging.info)
+    error_context.context("Get host numa topological structure", logging.info)
     timeout = float(params.get("login_timeout", 240))
     host_numa_node = utils_misc.NumaInfo()
     node_list = host_numa_node.online_nodes
     for node_id in node_list:
-        error.base_context("Bind qemu process to numa node %s" % node_id,
-                           logging.info)
+        error_context.base_context("Bind qemu process to numa node %s" % node_id,
+                                   logging.info)
         vm = "vm_bind_to_%s" % node_id
         params['qemu_command_prefix'] = "numactl --cpunodebind=%s" % node_id
         utils_memory.drop_caches()
+        node_MemFree = host_numa_node.read_from_node_meminfo(node_id, "MemFree")
```

Comment:
This returns str, let us typecast to int
`node_MemFree = int(host_numa_node.read_from_node_meminfo(node_id, "MemFree"))`